src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -83,7 +83,7 @@ public abstract class ClientApplicationBase
         /// Redirect Uri configured in the portal. Will have a default value. Not required, if the developer is using the
         /// default client Id.
         /// </summary>
-        public string RedirectUri { internal get; set; }
+        public string RedirectUri { get; set; }
```

Comment:
I am going to rename the folder from sample to something else. With the work we are doing these are not "samples" any more. They are more of a Dev Test Utility apps.